visidata/loaders/json.py

```diff
@@ -1,22 +1,64 @@
 import json
 
-from visidata import vd, date, VisiData, PyobjSheet, AttrDict, stacktrace, TypedExceptionWrapper, options, visidata, ColumnItem, wrapply, TypedWrapper, Progress, Sheet, InferColumnsSheet
+from visidata import vd, date, VisiData, PyobjSheet, AttrDict, stacktrace, TypedExceptionWrapper, options, visidata, ColumnItem, wrapply, TypedWrapper, Progress, Sheet, InferColumnsSheet, deduceType, SequenceSheet
 
 vd.option('json_indent', None, 'indent to use when saving json')
 vd.option('json_sort_keys', False, 'sort object keys when saving to json')
 vd.option('default_colname', '', 'column name to use for non-dict rows')
+vd.option('json_save_arrays', False, 'save table as rows of JSON arrays instead of JSONL dicts')
```

Comment:
We don't need these options, I think.  For saving, using the filetype `jsonla` should take care of it.  For loading, `open_jsonla` already checks the first line for an array type and falls back to `open_json` if it's not.  We'll make a better structure for guessing filetypes ultimately, but until then it should be workable without extra options.

Reply:
Removed.